orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -149,23 +148,25 @@ public APIGatewayProxyResponseEvent handleLogout(
         return generateLogoutResponse(uri, auditUser, clientId, rpPairwiseId);
     }
 
+    public APIGatewayProxyResponseEvent handleReauthenticationFailureLogout(
+            Session session,
+            APIGatewayProxyRequestEvent input,
+            String clientId,
+            URI errorRedirectUri) {
+        var auditUser = createAuditUser(input, session);
+        destroySessions(session);
```

Comment:
is there a reason we don't check for the existence of the sessions here?

Reply:
Ah if we're already doing a previous session exists check then I'm not too worried